webapp-django/crashstats/signature/static/signature/js/signature_report.js

```diff
@@ -334,6 +334,180 @@ $(function () {
         showComments();
     };
 
+    tabsLoadFunctions.graph = function() {
+
+        // Initialize the graph tab, bind all events and start loading
+        // default data.
+        var graphPanel = $('#graph-panel');
+        var contentElt = $('.content', graphPanel);
+        var loaderElt = $('.loader', graphPanel);
+        var selectElt = $('.channels-list', graphPanel);
+
+        // Get urls from graphPanel
+        var dataUrl = graphPanel.data('source-url');
+        var jsonUrl = graphPanel.data('jsonurl');
+
+        // Set up default params
+        var defaultChannel = 'nightly';
```

Comment:
That should not be hard-coded. Either use the already existing ``settings.CHANNELS`` (for example using ``settings.CHANNELS[0]``) or add a new setting key to hold that piece of data.